lib/blockchain_double.js

```diff
@@ -635,11 +635,12 @@ BlockchainDouble.prototype.processBlock = function(vm, block, commit, callback)
           receipt.gasUsed,
           result.createdAddress,
           receipt.status,
-          to.hex(result.bloom.bitvector)
+          to.hex(receipt.bitvector)
         );
         receipts.push(rcpt);
 
-        const rcptBuffer = Buffer.from(JSON.stringify(rcpt.toJSON()));
+        const rawReceipt = [receipt.status, receipt.gasUsed, receipt.bitvector, receipt.logs];
+        const rcptBuffer = utils.rlp.encode(rawReceipt);
```

Comment:
Is this just a preferred way to convert to buffer?

Reply:
The previous code was actually wrong and this corrects it. The variable name is wrong and I'll fix it. :-D